app/services/teacher_training_public_api/assign_site_attributes.rb

```diff
@@ -1,38 +0,0 @@
-module TeacherTrainingPublicAPI
```

Comment:
could those changes be in a separate consecutive PR, to allow easier reviewing/testing?